src/components/Cart/CartTotals.js

```diff
@@ -1,78 +1,67 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
-import { ProductConsumer } from '../Context';
 
 export default function CartTotals({ value }) {
-  const {
-    cartSubTotal,
-    cartTax,
-    cartTotal,
-    clearCart,
-    cartDiscount,
-    cartTotalAfterPromotion,
-  } = value;
```

Comment:
PS. it's actually a good practice to explicitly unwrap all the props that you need in this component.
this way you see all the props here right away, and you don't need to scan the code and look for `value.xyz` to learn what props you actually need